crepecake/src/main/java/com/google/cloud/tools/crepecake/image/Digest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.crepecake.image;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Represents a SHA-256 digest as defined by the Registry HTTP API v2 reference.
+ *
+ * @see <a
+ *     href="https://docs.docker.com/registry/spec/api/#content-digests">https://docs.docker.com/registry/spec/api/#content-digests</a>
+ */
+public class Digest {
```

Comment:
Since this is not a general purpose digest, perhaps name it a little more specifically. Maybe something like DockerDigest? DockerContentDigest? DockerRegistryDigest?

Reply:
Renamed to DescriptorDigest as it is the digest used in a content descriptor. LMKWYT about this naming, or perhaps ContentDigest might be better. We probably don't want to have it be a Docker*Digest since it will be used in eventually building an OCI image as well.